owlkettle.nimble

```diff
@@ -40,7 +40,7 @@ task genBook, "Generate the owlkettle nimibook book docs":
   echo "INITIALIZED NIMIBOOK"
 
   try:
-    exec "./nbook build --mm:refc --define:owlkettleNimiDocs "
+    exec "./nbook build --mm:refc --define:owlkettleNimiDocs --path:."
```

Comment:
This change was done for the same reason as before: It allows the nimibook to be built with *your clone* of the repo instead of the globally installed owlkettle package (if you have it installed, which is likely).